### ISSUE TITLE
Required Accessibility gatherer encountered an error: Cannot read property 'level' of undefined

### ISSUE THREAD

**Issue by amin212121** [GoogleChrome/lighthouse#12917]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

https://develop.priwatt.de/impressum/

### What happened?

We are running on the https://web.dev/ We are unable to get the Lighthouse SEO and Accessibility score.

Getting Error : Required Accessibility gatherer encountered an error: Cannot read property 'level' of undefined

![image](https://user-images.githubusercontent.com/36712833/129520972-9c6af09e-6139-4ee8-8b66-936da566c06c.png)

### What did you expect?

Will able to get the Lighthouse SEO and Accessibility metrics

### What have you tried?

I have tried running the lighthouse audit through normal chrome browser web page test tool without any issues.
Also, if I remove the GTM and facebook-pixel imports from function at the bottom, everything will work fine.
```
  window.chatLoaded = 0
  
  window.addEventListener("mousemove", loadChat)
  window.addEventListener("scroll", loadChat)
  window.addEventListener("touchstart", loadChat)
    
const loadChat = () => {
  if (!window.chatLoaded) {
    window.chatLoaded = 1
    addScript(`${process.env.GATSBY_HOME_URL}/supportboard/js/min/jquery.min.js`)
    setTimeout(() => addScript(`${process.env.GATSBY_HOME_URL}/supportboard/js/init.js`), 3000)

   import("react-facebook-pixel").then((ReactPixel) => {
      ReactPixel.default.init()
    })
   import("react-gtm-module").then((TagManager) => {
      TagManager.initialize({ gtmId: "some ID" })
    })
  } else {
    window.removeEventListener("mousemove", loadChat)
    window.removeEventListener("scroll", loadChat)
    window.removeEventListener("touchstart", loadChat)
  }
}
```
### How were you running Lighthouse?

 https://web.dev/measure/

### Lighthouse Version

Lighthouse 8.0.0

**Comment by patrickhulce** [comment]
Thanks for filing @amin212121! This was a bug in axe-core that [was fixed](https://github.com/dequelabs/axe-core/commit/4b7db3763735891972b8a13d6622fa30a687f3cb) and should no longer be happening in the latest version of Lighthouse (I cannot reproduce with v8.3.0).

web.dev should be updated sometime soon.
